tests/test_utils.py

```diff
@@ -979,33 +967,109 @@ def test_convert_params(input_: Any, expected: Any, convert_mode: Any) -> None:
                     "b": None,
                 },
             },
-            SimpleDataClass(a={"foo": 99}, b=[1, 99]),
-            id="simple",
+            (
+                SimpleDataClass(
```

Comment:
You seems to have dropped the simple case that was there initially.

Reply:
Now the `expected` test argument is the full instantiated config. It appears like it disappeared but it's still here.